spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -799,6 +799,9 @@ def write_data_spec(
 
         self._weight_scales[placement] = weight_scales
 
+    def clear_connection_cache(self):
+        self._retrieved_blocks = dict()
```

Comment:
peter says that static in structrual can change. and he also says his version with this merged in is still bust.......... which surprises me